lib/Provider/CustomOpenIDConnect.php

```diff
@@ -62,8 +63,19 @@ public function getUserProfile()
             if (empty($userProfile->email)) {
                 $userProfile->email = $profile->get('email');
             }
-            if (empty($userProfile->data['groups']) && $groups = $this->getGroups($profile)) {
```

Comment:
Same here

Reply:
So what would be the correct way of handling this?

Reply:
I resolve it by myself. Thanks for tip